tests/sles4sap/qesapdeployment/deploy.pm

```diff
@@ -25,7 +25,7 @@ sub run {
     upload_logs($inventory, failok => 1);
 
     # Set up azure native fencing
-    if (get_var('QESAPDEPLOY_FENCING') eq 'native' && $provider eq 'AZURE') {
+    if (get_var('QESAPDEPLOY_FENCING') eq 'native' && $provider eq 'AZURE' && !check_var('AZURE_FENCE_AGENT_CONFIGURATION', 'spn')) {
```

Comment:
Right setting name in this test is `QESAPDEPLOY_AZURE_FENCE_AGENT_CONFIGURATION`